src/firmware_new/boards/frankie_v1/stm32h7xx_hal_conf.h

```diff
@@ -98,9 +98,9 @@ extern "C"
  */
 #if !defined(HSE_VALUE)
 #define HSE_VALUE                                                                        \
-    ((uint32_t)25000000) /*!< Value of the External oscillator in Hz : FPGA case fixed   \
-                            to 60MHZ */
-#endif                   /* HSE_VALUE */
+    ((uint32_t)8000000) /*!< Value of the External oscillator in Hz : FPGA case fixed to \
+                           60MHZ */
```

Comment:
Is this still 60MHz?

Reply:
Yep, this is generated code, so even if it's wrong, not much we can do about it :grimacing: